src/Infrastructure/Services/FraudProtectionSettings.cs

```diff
@@ -21,6 +21,7 @@ public class FraudProtectionEndpoints
         public string Signup { get; set; }
         public string SignupStatus { get; set; }
         public string UpdateAccount { get; set; }
+        public string SignIn { get; set; }
```

Comment:
Do me a favor... in this file can you run ```ctrl+k+d``` to auto format and fix the tabs to spaces near the end of it?